pkg/model/deployment.go

```diff
@@ -332,6 +332,7 @@ func (b *BackstageDeployment) mountFilesFrom(containersFilter containersFilter,
 
 	volName := utils.GenerateVolumeNameFromCmOrSecret(objectName)
 	volSrc := corev1.VolumeSource{}
+	//nolint:staticcheck // ignore QF1003 for better readability
```

Comment:
Same point here: a tagged switch might improve readability, IMO. Have you tried it before disabling it?